io.catenax.specialCharacteristics.measurement/1.0.0/Measurement.ttl

```diff
@@ -0,0 +1,63 @@
+
+
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.1.0#> .
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.1.0#> .
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.1.0#> .
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.1.0#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix : <urn:samm:io.catenax.specialCharacteristics.measurement:1.0.0#> .
+@prefix bpn: <urn:samm:io.catenax.shared.business_partner_number:2.0.0#> .
+@prefix samm-u: <urn:samm:org.eclipse.esmf.samm:unit:2.1.0#> .
+@prefix serialPart: <urn:samm:io.catenax.serial_part:3.0.0#> .
+
+:Measurement a samm:Aspect ;
+   samm:preferredName "SpecialCharacteristicMeasurment"@en ;
+   samm:description "Contains the Measurment for a special characteristic"@en ;
+   samm:properties ( serialPart:localIdentifiers bpn:bpnlProperty :customerPartId :characteristicId :results [ samm:property :measurementType; samm:optional true ] :revisionIndex ) ;
+   samm:operations ( ) ;
+   samm:events ( ) .
+
+:customerPartId a samm:Property ;
+   samm:preferredName "customerPartId"@en ;
+   samm:description "This number is the identification of the Part"@en ;
+   samm:characteristic :CharacteristicCustomerPartID ;
+   samm:exampleValue "BPN0000000" .
+
+:characteristicId a samm:Property ;
+   samm:preferredName "characteristicId"@en ;
+   samm:description "Identification of the special characteristic in in relation of the specified part"@en ;
+   samm:characteristic :CharacteristicCustomerPartID ;
+   samm:exampleValue "S1" .
+
+:results a samm:Property ;
```

Comment:
The property  **results** is currently defined as a list of objects. However, the objects contained therein are not further defined. 

I'm against the definition of an “open” list of objects. Otherwise, there are no rules which objects are stored here, so that everything possible can be implemented and therefore the purpose of standardization becomes obsolete.

Reply:
It was a explicit decision to keep it "open" because the set of characteristic is such a heterogenous set that one would have to create almost everytime a new object to fully capture the requirements for the measurement. I`ll raise to and discuss this with Jochen and Sebastian .

Reply:
@LuisRickert I got your point of view and agree. But from a semantic perspective, it is **not recommended** to define an aspect model with an open object list. If we publish the model in this form, there won't be rules which data in which form are exchanged between business partners, as the data list is not further defined and therefore not standardized. This is **in contrary to the Catena-X approach**!

### Suggestion
As mentioned above, I recommend **defining all known and possible properties/attributes** for the object list for the version and expanding the aspect model with new data in the future as needed.

Reply:
@johannsvarela after discussing this with Jochen and Sebastian we decided to keep the model as it is, since there is no sane way to provide a structure that could support a wide variety of measurements. We are aware that this can result in the rejection of the model.

Reply:
Hi @johannsvarela, @LuisRickert ,
there is one more important reason why we created the result as a set. The result is product specfic and not general usable for all product. Therefore no general aspect model for products in general possible.  There is also no traceability through multiple tiers possible, because every special characteristic is linked to one specific product.